extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Utility functions for running Optimizely A/B tests with VisualEditor
+ */
+
+( function () {
+	/**
+	 * @returns {number} The A/B test variant number for the Toolbar A/B test
+	 */
+	ve.init.wikia.getToolbarABTestVariantNumber = function () {
```

Comment:
Sure, but I just want you to aware. optimizely experiment might be activated right away on the page view - when user opens the page - or later with JavaScript - when user for instance opens visualeditor.

so in your case you will get more many users bucketed than users actually using visualeditor and optimizely perhaps will report that received data is conclusive before it really is.

and by activating I mean assigning to particular bucket/group.

it seems like you optimizely only for bucketing users, but not for tracking "success" / conversion ratio. just checking if that's the intention.